VadeSecure/vade_secure_m365/_meta/fields.yml

```
@@ -58,6 +58,11 @@ vadesecure.attachments:
   short: vadesecure.to_header
   type: array
 
+vadesecure.auth_results_details:
+  description: Details of security protocols, mostly SPF, DKIM and DMARC.
+  name: vadesecure.auth_results_details
+  type: object
+
```

Can you split this field into subfields?

```suggestion
vadesecure.auth_results_details.spf:
  description: The result of the Sender Policy Framework (SPF)
  name: vadesecure.auth_results_details.spf
  type: keyword

vadesecure.auth_results_details.dkim:
  description: The result of the DomainKeys Identified Mail (DKIM)
  name: vadesecure.auth_results_details.dkim
  type: keyword

vadesecure.auth_results_details.dmarc:
  description: Result of the Domand-based Message Authentication Reporting and Conformance (DMARC)
  name: vadesecure.auth_results_details.dmarc
  type: keyword

```